craftship/codebox-npm
# Add npm deprecate Support

## What did you implement:

Adds `npm deprecate` support to registry.

## How did you implement it:

* Update json sent from request body coming through to put package endpoint.

## How can we verify it:

* `npm t`
* Deploy registry publish a couple of package then deprecate a version
* If you then install that version you should see deprecation warning

## Todos:

***Is this a breaking change?:*** NO
